gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Driver/MessagesTests.cs

```diff
@@ -119,7 +119,8 @@ public async Task ShouldUseSpecifiedEvaluationTimeout()
                 evaluationStopWatch.Stop();
                 Assert.Contains("ServerTimeout", thrownException.Message);
                 Assert.Contains(timeOutInMs.ToString(), thrownException.Message);
-                Assert.True(evaluationStopWatch.ElapsedMilliseconds < scriptSleepTimeInMs);
+                Assert.True(evaluationStopWatch.ElapsedMilliseconds < scriptSleepTimeInMs,
+                    $"Elapsed {evaluationStopWatch.ElapsedMilliseconds} > {scriptSleepTimeInMs}");
```

Comment:
Was this added for debugging?

Reply:
this is the standard way to get a better error description. Message `Elapsed ...` will be printed only if Assert failed